challenges/objects-arrays.js

```diff
@@ -108,14 +149,20 @@ The zoos are concerned about animals with a lower population count. Using filter
 
 */
 const lowPopulationAnimals = [];
+zooAnimals.filter(animal => {
+  if (animal.population < 5){
+    lowPopulationAnimals.push(animal);
+  }
+});
 console.log(lowPopulationAnimals);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States. Find the total population from all the zoos using the .reduce() method. Remember the reduce method takes two arguments: a callback (which itself takes two args), and an initial value for the count.
 
 */
-const populationTotal = 0;
+const populationTotal = zooAnimals.reduce((total, {population}) => total + population, 0);
+
 console.log(populationTotal);
 
```

Comment:
awesome work on these! just don't forget that the starting index in your .reduce should be outside of the curly braces, like this:
```
const populationTotal = zooAnimals.reduce((accumulator, currentItem) =>{
  return accumulator + currentItem.population
}, 0)